store/tikv/oracle/oracles/pd.go

```diff
@@ -270,14 +270,26 @@ func (o *pdOracle) GetLowResolutionTimestampAsync(ctx context.Context, opt *orac
 	}
 }
 
-func (o *pdOracle) getStaleTimestamp(txnScope string, prevSecond uint64) (uint64, error) {
+func (o *pdOracle) getStaleTimestamp(ctx context.Context, txnScope string, prevSecond uint64) (uint64, error) {
 	ts, ok := o.getLastTS(txnScope)
 	if !ok {
-		return 0, errors.Errorf("get stale timestamp fail, invalid txnScope = %s", oracle.GlobalTxnScope)
+		// If we didn't find lastTS, try to directly request tso
+		ts, err := o.getTimestamp(ctx, txnScope)
+		if err != nil {
+			return 0, err
+		}
+		o.setLastTS(ts, txnScope)
+		return 0, errors.Errorf("get stale timestamp fail, txnScope: %s", txnScope)
```

Comment:
revise the misleading error msg.

Reply:
Would it be better to move the fallback to `GetStaleTimestamp`, and leave `getStaleTImestamp` unchanged(except this error message)?

Reply:
it's ok to me, updated.